### PR TITLE
SDI-358 Offender Events UI - allow single pod to recycle

### PR DESCRIPTION
**Pull request by steverendell**
Just set the PDB to allow single pod to recycle

### CODE REVIEW

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ---
 # Environment specific values, override helm_deploy/court-register/values.yaml
 generic-service:
-  replicaCount: 2
+  replicaCount: 1
   ingress:
-    host: offender-events-ui-dev.prison.service.justice.gov.uk
+    hosts:
```
Comment (petergphillips) [inline helm_deploy/values-dev.yaml:6]: not a great fan of this change - the helm chart allows either so think just specifying the `host` seems simpler.

**Review by petergphillips — DISMISSED** [review]
LGTM

File: helm_deploy/values-dev.yaml (hunk 1 above)
Comment (steverendell) [reply helm_deploy/values-dev.yaml:6]: ok, i was thinking "host" may be deprecated, but will revert 👍

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**